Dockerfile

```diff
@@ -1,33 +1,33 @@
 FROM golang:1.19-alpine AS build
-ARG GOTESTWAF_VERSION="unknown"
+RUN apk --no-cache add git
 WORKDIR /app/
 COPY . .
-RUN go build -ldflags "-X github.com/wallarm/gotestwaf/internal/version.Version=${GOTESTWAF_VERSION}" \
+RUN go build -ldflags "-X github.com/wallarm/gotestwaf/internal/version.Version=$(git describe)" \
 		-o gotestwaf ./cmd/
 
 FROM alpine
 WORKDIR /app
 RUN apk add --no-cache chromium fontconfig
 RUN apk add --no-cache wget curl && \
```

Comment:
If I understand that correctly, these two `apk add` commands are separated to get benefits from docker cache for the first one (e.g. not to download chromium and fontconfig if they are already in cached layer), am I right? Or not? Otherwise, if I'm wrong, they can be combined.

Reply:
Fixed - https://github.com/wallarm/gotestwaf/pull/160/commits/714c39112a29e1a8eefc3e97fc33ef9fe59bd0b9